ang/civiawards/dashboard/directives/dashboard-action-separator.js

```diff
@@ -0,0 +1,26 @@
+(function (angular, _) {
+  var module = angular.module('civiawards');
+
+  module.controller('DashboardActionSeparatorController', DashboardActionSeparatorController);
+
+  /**
+   * Handles the visibility and click event for the "Add Award" dashboard action button.
+   *
+   * @param {object} $scope scope object
+   * @param {object} $location the location service
+   */
+  function DashboardActionSeparatorController ($scope, $location) {
+    $scope.isVisible = isVisible;
+
+    /**
+     * Is only visible on the Awards dashboard.
+     *
+     * @returns {boolean} true when case type category url param is awards
+     */
+    function isVisible () {
+      var urlParams = $location.search();
+
+      return urlParams.case_type_category === 'awards';
+    }
+  }
+})(angular, CRM._);
```

Comment:
I think we need to change the visibility approach here to avoid implementing the same logic for multiple buttons.

- `awards-dashboard-action-items.config.js` should add these buttons conditionally only if the `case_type_category` param is equal to `awards`.
- `dashboard-action-separator.html` should have no `ng-controller` or `ng-show`.
- Delete `dashboard-action-separator.js`.
- Remove the `isVisible` method from `more-filters-dashboard-action-button.controller.js` and `add-award-dashboard-action-button.controller.js`

Reply:
As discussed, removed the separator.